pkg/wallet/wallet.go

```diff
@@ -11,31 +11,38 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-
-	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
+	"time"
 
 	"github.com/btcsuite/btcutil/base58"
-	"golang.org/x/crypto/nacl/box"
-
 	"github.com/hyperledger/aries-framework-go/pkg/didcomm/crypto"
 	"github.com/hyperledger/aries-framework-go/pkg/didcomm/crypto/jwe/authcrypt"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
 	"github.com/hyperledger/aries-framework-go/pkg/storage"
+	"golang.org/x/crypto/nacl/box"
+)
+
+const (
+	didFormat    = "did:%s:%s"
+	didPKID      = "%s#keys-%d"
+	didServiceID = "%s#endpoint-%d"
+	didContext   = "https://w3id.org/did/v1"
```

Comment:
nitpick: `didContext` should be defined in the `did` package.

Reply:
done